src/styleRe.rei

```diff
@@ -1,5 +1,14 @@
 type t;
 
+/** Coerces an array of styles to a single style.
```

Comment:
Let's use simple vocabulary in comments. How about `Generates a style out of array of styles` or something?